rebasehelper/checkers/rpmdiff_tool.py

```diff
@@ -55,6 +57,51 @@ def get_checker_name(cls):
     def is_default(cls):
         return cls.DEFAULT
 
+    @classmethod
+    def get_license_changes(cls):
+        """
+        Finds out if there is a possibility of license change in new sources
+
+        :return: true if license changes probably happened
+        """
+        diffs = []
+        for srcs in ['new_sources', 'old_sources']:
+            srcs = os.path.join(cls.workspace_dir, srcs)
+
+            # Get all files
+            files = PathHelper.find_all_files(srcs, '*')
```

Comment:
What is this for? The list of changed files is in `cls.results_dict['changed']`, right?

Reply:
You are right, but `cls.results_dict['changed']` contains system paths and not `rebase-helper-workspace` paths.

Reply:
Right, because rpmdiff compares binary rpms. That means licensecheck should be run against contents of those binary rpms as well. You can't assume that a source file named X will have the same name (and more importantly, same content) after build.